ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/DistributingExecutor.java

```diff
@@ -159,4 +176,20 @@ private void checkAuthorization(
       throw new KsqlServerException("The KSQL server is not permitted to execute the command", e);
     }
   }
+
+  private void throwIfInsertOnReadOnlyTopic(
```

Comment:
Did you see Vic's comment about adding the config(s) to the list of immutable configs? https://github.com/confluentinc/ksql/pull/4322#discussion_r369327471